test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTable/run.csx

```diff
@@ -0,0 +1,22 @@
+﻿#r "..\..\..\Microsoft.Azure.ApiHub.Sdk.dll"
+
+using System;
+using Microsoft.Azure.ApiHub.Sdk.Table;
+
+public class SampleEntity
+{
+    public int Id { get; set; }
+    public string Text { get; set; }
+}
+
+public static async Task Run(string text, ITable<SampleEntity> table, TraceWriter log)
```

Comment:
Looks like you're only testing C# currently, not the codepaths used for other languages